measures/BuildExistingModel/measure.rb

```diff
@@ -266,17 +295,49 @@ def run(model, runner, user_arguments)
     measures['HPXMLtoOpenStudio'][0]['debug'] = args['debug'].get if args['debug'].is_initialized
     measures['HPXMLtoOpenStudio'][0]['add_component_loads'] = args['add_component_loads'].get if args['add_component_loads'].is_initialized
 
-    if not apply_measures(hpxml_measures_dir, { 'BuildResidentialHPXML' => measures['BuildResidentialHPXML'], 'BuildResidentialScheduleFile' => measures['BuildResidentialScheduleFile'], 'HPXMLtoOpenStudio' => measures['HPXMLtoOpenStudio'] }, new_runner, model, true, 'OpenStudio::Measure::ModelMeasure', 'existing.osw')
-      new_runner.result.warnings.each do |warning|
-        runner.registerWarning(warning.logMessage)
+    # Specify measures to run
+    if run_hescore_workflow
+      measures['BuildResidentialHPXML'][0]['apply_defaults'] = true
+      measures_hash = { 'BuildResidentialHPXML' => measures['BuildResidentialHPXML'] }
+    else
+      measures_hash = { 'BuildResidentialHPXML' => measures['BuildResidentialHPXML'], 'BuildResidentialScheduleFile' => measures['BuildResidentialScheduleFile'], 'HPXMLtoOpenStudio' => measures['HPXMLtoOpenStudio'] }
+    end
+
+    if not apply_measures(hpxml_measures_dir, measures_hash, new_runner, model, true, 'OpenStudio::Measure::ModelMeasure', 'existing.osw')
+      register_logs(runner, new_runner)
+      return false
+    end
+
+    # Run HEScore Measures
+    if run_hescore_workflow
+      measures['HEScoreHPXML'] = [{}]
+      measures['HEScoreHPXML'][0]['hpxml_path'] = hpxml_path
+      measures['HEScoreHPXML'][0]['output_path'] = File.expand_path('../hes.json')
+      measures['HEScoreRuleset'] = [{}]
+      measures['HEScoreRuleset'][0]['json_path'] = '/var/simdata/openstudio/run/hes.json'
+      measures['HEScoreRuleset'][0]['hpxml_output_path'] = '/var/simdata/openstudio/run/hes.xml'
+      measures['HPXMLtoOpenStudio'][0]['hpxml_path'] = measures['HEScoreRuleset'][0]['hpxml_output_path']
+
+      # HEScoreHPXML
+      measures_hash = { 'HEScoreHPXML' => measures['HEScoreHPXML'] }
+      if not apply_measures(measures_dir, measures_hash, new_runner, model, true, 'OpenStudio::Measure::ModelMeasure')
```

Comment:
Why not store HEScoreHPXML on the OS-HEScore repo so that you could combine `apply_measures` calls for HEScoreHPXML and HEScoreRuleset?

Reply:
@nmerket Thoughts on moving HEScoreHPXML to OS-HEScore? Would package the HES measures together a little better

Reply:
Are there any drawbacks to moving the measure? I agree that it does seem more logical to have the measure in the OS-HEScore repo, great comment @joseph-robertson. For example, in the future there could be another application (other than ResStock) that wants to use this measure.

Reply:
And who knows, maybe someday HEScore _itself_ would want to use this measure as part of our OS-HEScore workflow.

Reply:
I think it's a great idea.